job.go

```diff
@@ -16,7 +17,8 @@ import (
 type Job struct {
 	mu *jobMutex
 	jobFunction
-	interval          int             // pause interval * unit between runs
+	interval          int             // interval * unit between runs
+	random                            // details for randomness
```

Comment:
Maybe better to add it as `random random // details for randomness` because otherwise we can use fields from `random struct` directly from `job struct`.